packages/pixeloven-webpack/config/src/index.ts

```diff
@@ -138,10 +174,38 @@ function getConfig(options: Options) {
              * @env development
              */
             ifDevelopment(new CaseSensitivePathsPlugin()),
-            // new CircularDependencyPlugin({
-            //     // exclude detection of files based on a RegExp
-            //     exclude: /node_modules/,
-            // }),
+            /**
+             * Allows for circular dependency detection
+             *
+             * @todo should determine why we can't just push errors/strings to compilation.warnings
+             */
+            new CircularDependencyPlugin({
+                // exclude detection of files based on a RegExp
```

Comment:
Should we extract these options? They look like they're the same as above.

Reply:
I plan on merging the two plugin arrays. I actually can't stand the shared helpers but did that only so i could transition to this more unified config. The PR to support libraries started this work I just needed to wrap this stuff up and then get back to it.